orbstation/antagonists/traitor/objectives/kidnapping.dm

```diff
@@ -0,0 +1,284 @@
+
+	addtimer(CALLBACK(src, .proc/handle_victim, sent_mob), 1.5 SECONDS)
+
+	if(sent_mob != victim || sent_mob.stat == DEAD)
+		fail_objective(penalty_cost = telecrystal_penalty)
+		source.startExitSequence(source)
+		return
+
+	succeed_objective()
+	source.startExitSequence(source)
+
+/datum/traitor_objective/kidnapping/proc/handle_victim(mob/living/carbon/human/sent_mob)
+	addtimer(CALLBACK(src, .proc/return_victim, sent_mob), 3 MINUTES)
+	if(sent_mob.stat == DEAD)
+		return
+
+	sent_mob.flash_act()
+	sent_mob.adjust_timed_status_effect(10 SECONDS, /datum/status_effect/confusion)
+	sent_mob.adjust_timed_status_effect(10 SECONDS, /datum/status_effect/dizziness)
+	sent_mob.blur_eyes(5 SECONDS)
+	to_chat(sent_mob, span_hypnophrase(span_reallybig("A million voices echo in your head... <i>\"Your mind held many valuable secrets - \
+		we thank you for providing them. Your value is expended, and you will be ransomed back to your station. We always get paid, \
+		so it's only a matter of time before we ship you back...\"</i>")))
+
+/datum/traitor_objective/kidnapping/proc/return_victim(mob/living/carbon/human/sent_mob)
+	if(!sent_mob || QDELETED(sent_mob)) //suicided and qdeleted themselves
+		return
+
+	var/list/possible_turfs = list()
+	for(var/turf/open/open_turf in dropoff_area)
+		if(open_turf.is_blocked_turf() || isspaceturf(open_turf))
+			continue
+		possible_turfs += open_turf
+
+	if(!LAZYLEN(possible_turfs))
+		var/turf/new_turf = get_safe_random_station_turf()
+		if(!new_turf) //SOMEHOW
+			to_chat(sent_mob, span_hypnophrase(span_reallybig("A million voices echo in your head... <i>\"Seems where you got sent here from won't \
+				be able to handle our pod... You will die here instead.\"</i></span>")))
+			if (sent_mob.can_heartattack())
+				sent_mob.set_heartattack(TRUE)
+			return
+
+		possible_turfs += new_turf
+
+	var/obj/structure/closet/supplypod/return_pod = new()
+	return_pod.bluespace = TRUE
+	return_pod.explosionSize = list(0,0,0,0)
+	return_pod.style = STYLE_SYNDICATE
+
+	do_sparks(8, FALSE, sent_mob)
+	sent_mob.visible_message(span_notice("[sent_mob] vanishes!"))
+	for(var/obj/item/belonging in sent_mob)
+		if(belonging == sent_mob.get_item_by_slot(ITEM_SLOT_ICLOTHING) || belonging == sent_mob.get_item_by_slot(ITEM_SLOT_FEET))
+			continue
+
+		sent_mob.transferItemToLoc(belonging)
+
+	for(var/obj/item/belonging in victim_belogings)
+		belonging.forceMove(return_pod)
+
+	sent_mob.forceMove(return_pod)
+	sent_mob.flash_act()
+	sent_mob.adjust_timed_status_effect(10 SECONDS, /datum/status_effect/confusion)
+	sent_mob.adjust_timed_status_effect(10 SECONDS, /datum/status_effect/dizziness)
+	sent_mob.blur_eyes(5 SECONDS)
+
+	to_chat(sent_mob, span_hypnophrase("Your mind is a blur... you have no memory of your kidnapper or your time spent in captivity."))
```

Comment:
Is it necessary to include "and your time spent in captivity"? 
It seems the crew is informed that there was a kidnapping anyway, so it's not a big secret what happened to you.

Reply:
I just thought it wouldn't hurt to be thorough, especially since if, by some happenstance, multiple people are kidnapped at the same time, they both show up in the same holding area and can interact with each other.

Reply:
I guess to me that sounds like more reason to remove it, it would be a shame to hang out with another prisoner and then have to forget about it.  

Could you add the “you don’t remember how you got here” text to appearing in the holding area instead?